### PR TITLE
Make SONiC the default flavor.

### LINKED ISSUE #154: Make SONiC the default flavor
Should be possible after #153.

### PR DESCRIPTION
**Pull request by Gerrit91**
Closes #154.

### CODE REVIEW

**Comment by robertvolkmann** [comment]
@Gerrit91 sonic-vs.img is missing on the integration platform.

**Comment by robertvolkmann** [comment]
@Gerrit91 What is your opinion to use both flavors for cleanup? The leaves of one flavor cannot be removed by the other flavor. 
```
MINI_LAB_FLAVOR=cumulus make cleanup
MINI_LAB_FLAVOR=sonic make cleanup
```

**Comment by Gerrit91** [comment]
Not yet decided, maybe we can just destroy for all flavors in cleanup.

File: Makefile [hunk 1]
```diff
@@ -131,8 +131,13 @@ cleanup-control-plane:
 .PHONY: cleanup-partition
 cleanup-partition:
 	mkdir -p clab-mini-lab
+
 	sudo $(CONTAINERLAB) destroy --topo mini-lab.cumulus.yaml
+
+# destroying the sonic lab requires the image to exist, otherwise it fails with bind path verification
+	touch sonic-vs.img
 	sudo $(CONTAINERLAB) destroy --topo mini-lab.sonic.yaml
+	rm -f sonic-vs.img
```
Comment (robertvolkmann) [inline Makefile]: Using an conditional?
```
ifneq ($(wildcard sonic-vs.img),)
	sudo $(CONTAINERLAB) destroy --topo mini-lab.sonic.yaml
endif
```

Comment (robertvolkmann) [reply Makefile]: Maybe its easier to create a container image that contains the VM image. Let me try something next week.

Comment (Gerrit91) [reply Makefile]: Have you come up with something better? Release pipeline is currently broken due to cleanup problems so we should find something soon.

Comment (robertvolkmann) [reply Makefile]: I will look into it during my journey.

**Review by ulrichSchreiner — APPROVED** [review]

File: Makefile (hunk 1 above)
Comment (Gerrit91) [reply Makefile]: We prioritized fixing the pipeline for now. If you find some time to improve the cleanup please feel free to raise a new pull request.

**Pull request merged**